app/views/demographic_questions/_form.html.erb

```diff
@@ -11,6 +11,11 @@
     </div>
   <% end %>
 
+  <div class="field">
+    <%= form.label :question_type %>
+    <%= form.collection_select :question_type, demographic_question_types, :to_s, :to_s, prompt: true %>
```

Comment:
Nice, let's actually go back and implement this on the other place we're doing it. This is a much better method

Reply:
Thanks what is the other place? Do you refer to the responses form?

Reply:
Sure